.github/workflows/build.yml

```diff
@@ -65,11 +65,12 @@ jobs:
       - name: "Verify no plugin issues"
         run: mvn artifact:check-buildplan --batch-mode --no-transfer-progress
 
-      - name: "Verify reproducible build"
+      - name: "Do clean install of dependencies"
         # See https://maven.apache.org/guides/mini/guide-reproducible-builds.html#how-to-test-my-maven-build-reproducibility
         run: |
           mvn clean install --batch-mode --no-transfer-progress -Dproguard.skip -DskipTests
           # Run with `-Dbuildinfo.attach=false`; otherwise `artifact:compare` fails because it creates a `.buildinfo` file which
           # erroneously references the existing `.buildinfo` file (respectively because it is overwriting it, a file with size 0)
           # See https://issues.apache.org/jira/browse/MARTIFACT-57
```

Comment:
Since these `# Run with ...` comments are only relevant for the `mvn` command below, it would be good to move them down (and adjust their indentation), so that they are between the `name` and the `run` below.

Reply:
Ah yes moved them